FloatingTextField/FloatingTextField.swift

```diff
@@ -27,47 +27,47 @@ class FloatingTextField: UITextField, UITextFieldDelegate {
     
     @IBInspectable var placeHolder: String = ""
     @IBInspectable var placeHolderFont: UIFont?
-    @IBInspectable var PlaceHolderColor:UIColor = UIColor.lightGray
-    @IBInspectable var PlaceHolderColorAfterText:UIColor = UIColor.black
-    @IBInspectable var PlaceHolderBackgroundColor:UIColor = UIColor.white
-    @IBInspectable var PlaceHolderLeftPadding: CGFloat = 15
-    @IBInspectable var PlaceHolderLeftPaddingAfterAnimation: CGFloat = 15
+    @IBInspectable var placeHolderColor:UIColor = UIColor.lightGray
```

Comment:
There's nothing inherently wrong with using this naming scheme, it's just not standard. 

"Standard" is a dangerous word because there are many ways to code and different people feel comfortable with different styles. However, there are conventions that are accepted by the majority of the community and so you should at least try to follow them. Being original isn't about breaking accepted norms, it's about creative solutions to problems.